tests/bdd/common/kubectl_mayastor.py

```diff
@@ -0,0 +1,39 @@
+import logging
+import os
+import subprocess
+
+
+def get_bin_path():
+    bins = os.getenv("BIN")
+    if bins:
+        return os.path.join(bins, "kubectl-mayastor")
+    logging.warning(f"Environmental variable 'BIN' is not set")
+    return "kubectl-mayastor"
+
+
+def kubectl_mayastor(args: str):
+    command = [
+        "/bin/bash",
+        "-c",
```

Comment:
can you not run the bin path directly?

Reply:
Yes, I think running it directly would work.